valmlord/gulpify
# fix: redefined pug folder structure

## Description
This pull request fixes the file structure in the build directory for Pug files, ensuring that the output is organized correctly.

## Changes
- Updated `gulpfile.js` to modify the build process for Pug files.
- Adjusted the output paths for Pug templates to ensure they are placed in the correct directories within the `build` folder.

## How to Test
2. Place Pug template files in the `app/pug` directory.
3. Execute the `gulp buildHTML` command.
4. Verify that the compiled HTML files are correctly structured in the `build` directory.

## Additional Notes
- This update aims to correct the file organization in the build directory, improving the overall project structure.
